src/subscription_manager/scripts/rhsmcertd_worker.py

```diff
@@ -45,6 +46,10 @@
     from subscription_manager.cp_provider import CPProvider
 
 
+DEFAULT_AUTOREGISTER_INTERVAL = 60
+DEFAULT_AUTOREGISTER_IDENTITY_INTERVAL = 60 * 60
+
```

Comment:
Since `auto_registration_identity_interval` in config file is now 10 minutes, shouldn't be this just `60*10`?